src/pages/trade-types/options/index.tsx

```diff
@@ -21,9 +21,9 @@ const meta_attributes = {
 }
 
 const Options = () => {
-    const { is_eu_country } = React.useContext(DerivStore)
+    const { is_eu, is_uk } = getCountryRule()
 
-    return !is_eu_country ? (
+    return !is_eu || !is_uk ? (
```

Comment:
For this one we can use `is_row` for Rest of the world except UK & EU

Reply:
oh cool, let me change it.